topics/statistics/tutorials/machinelearning/slides.html

```diff
@@ -0,0 +1,73 @@
+---
+layout: tutorial_slides
+topic_name: statistics
+tutorial_name: machinelearning
+logo: "GTN"
+---
+
+### Machine learning
+
+![ml basics pic](images/ml_basics.png)
+
```

Comment:
would you mind adding some speaker notes to the slides? especially for the ones with just pictures on them. Basically just write down what you would say when you present this. It will help others used these materials :)

speaker note syntax is:

```
### Machine learning

![ml basics pic](images/ml_basics.png)

???
some notes about the slides

---

```

Reply:
You can see more examples of the available features in our dedicated [tutorial on how to write slides](https://galaxyproject.github.io/training-material/topics/contributing/tutorials/create-new-tutorial-slides/slides.html)